recipes/flann/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# cannot build flann from within the source directory
+mkdir build
+cd build
+
+# On OSX, we need to ensure we're using conda's gcc/g++
+if [[ `uname` == Darwin ]]; then
+    export CC=gcc
+    export LD=gcc
+    export CXX=g++
+fi
+
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX -DBUILD_MATLAB_BINDINGS:BOOL=OFF -DBUILD_PYTHON_BINDINGS:BOOL=OFF -DBUILD_EXAMPLES:BOOL=OFF
+
+make install
```

Comment:
Seems like FLANN is install its docs, which is a bit unnecessary if there is an option to disable this that would work. Alternatively, we can just add `rm -rf "${PREFIX}/share` as all that is there is a beefy PDF.